sdk/cosmos/azure-cosmos/azure/cosmos/_cosmos_client_connection.py

```diff
@@ -1029,7 +1024,7 @@ def QueryItems(
         database_or_container_link: str,
         query: Optional[Union[str, Dict[str, Any]]],
         options: Optional[Mapping[str, Any]] = None,
-        partition_key: Optional[Union[str, float, bool]] = None,
+        partition_key: Optional[Union[str, float, bool, List[Union[str, float, bool]]]] = None,
```

Comment:
The corresponding docstring would also need an update here :)

`:param str or float or bool or None or list[str] or list[float] or list[bool]